Remotion/Mixins/Core/MixinTypeUtility.cs

```diff
@@ -38,6 +38,8 @@ public static class MixinTypeUtility
     private static readonly ConcurrentDictionary<Type, ReadOnlyCollection<Type>> s_exactMixinTypesCache =
         new ConcurrentDictionary<Type, ReadOnlyCollection<Type>>();
 
+    private static readonly ConcurrentDictionary<ValueTuple<Type, Type>, bool> s_mixinTypeCache = new();
```

Comment:
We should make this named, too.
```suggestion
    private static readonly ConcurrentDictionary<(Type ConfiguredMixinType, Type MixinType), bool> s_mixinTypeCache = new();
```